File: clients/javascript/src/models/getNamespaceIn.ts
// this file is @generated





export interface GetNamespaceIn {
    name: string;
}

export const GetNamespaceInSerializer = {
    _fromJsonObject(object: any): GetNamespaceIn {
        return {
            name: object['name'],
            };
    },

    _toJsonObject(self: GetNamespaceIn): any {
        return {
            'name': self.name,
            };
    }
}

File: clients/rust/src/models/get_namespace_in.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct GetNamespaceIn {
    pub name: String,
}

impl GetNamespaceIn {
    pub fn new(name: String) -> Self {
        Self { name }
    }
}
